src/game/reducers/update-hand/index.ts

```diff
@@ -0,0 +1,18 @@
+import { IGame, IPlayer } from '../../types'
+
+export const updateHand = (
+  game: IGame,
+  playerId: IPlayer['id'],
+  hand: IPlayer['hand']
+): IGame => {
+  return {
```

Comment:
same comment here as in `updateDiscardPile` about all the destructuring

Reply:
This is addressed in the new meta-reducer that abstracts this and `updateDiscardPile` in the next PR: https://github.com/jeremyckahn/farmhand-shuffle/commit/f9c147717492a6a5c3f00b5e249d3ac30e9d6100